src/main/webapp/index.html

```diff
@@ -20,15 +21,18 @@
       <div id="form-title">
         <p>Choose Type of Input</p>
       </div>
+      <label><input type="checkbox" onclick="setLocationCookie()" /> Use Location</label>
+      <p id="location-warning"></p>
```

Comment:
Also, this area doesn't exist because an alert is sent. It seems like that didn't get merged in.